src/vm/stackwalk.cpp

```diff
@@ -725,7 +723,7 @@ PCODE Thread::VirtualUnwindNonLeafCallFrame(T_CONTEXT* pContext, KNONVOLATILE_CO
 #if defined(_WIN64)
     UINT64              EstablisherFrame;
```

Comment:
Yes, the _WIN64  is equivalent to BIT64. It would be nice to clean that up in the codebase at some point. There are about 600 usages in 180 files of the _WIN64.
But for now, it would not hurt if you could change it to BIT64 in place you are changing (please note it is BIT64, not _BIT64).